### PR TITLE
feat: Header

### PR DESCRIPTION
**Pull request by markshenouda**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.Any change needs to be discussed before proceeding.**

## Closing issues

closes #ISSUE_NUMBER_GOES_HERE

...

## Description

...

## Changes

- High level
- changes that
- you made
- ...

### CODE REVIEW

File: src/components/Header/Nav/ProductNavigation/Desktop/ProductNavigation.tsx [hunk 1]
```diff
@@ -45,31 +40,38 @@ export const ProductNavigation = ({ path, setNavMenuOpen, productsNav, subProduc
...
           <Item>
-            <RadixTrigger className="nav-subproduct" ref={subProductMenuRef}>
-              <Trigger icon={icon} label={label} />
-            </RadixTrigger>
-            {(subProductTrigger || label === "Resources") && subProductsNav && (
-              <RadixContent className={styles.content}>
-                <SubProductContent subProductsNav={subProductsNav} />
-              </RadixContent>
-            )}
+            <span
```
Comment (Zelig880) [inline src/components/Header/Nav/ProductNavigation/Desktop/ProductNavigation.tsx:43]: With your change this turned into a span. We need to ensure that it is correctly set for accessibility:
- Give it a role of button
- Set it an "aria-expanded" attribute
- Make sure the attribute is set to true when the dropdown opens

File: src/components/Header/Nav/ProductNavigation/Desktop/MegaMenu.module.css [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  background-color: white;
+  height: fit-content;
+  border-radius: 0.5rem;
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+  overflow: hidden;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
+}
+
+.wrapper::before {
```
Comment (Zelig880) [inline src/components/Header/Nav/ProductNavigation/Desktop/MegaMenu.module.css]: Not sure if this is the little Triangle on top o the menu, but that is not aligning properly. I tried on smaller and larger screens and the triangle does not fit properly. Maybe a simple solution would be to actually let that be part of the "button" and not the menu itself.

Let me know if I can help brainstorm

File: src/components/Header/Nav/ProductNavigation/Mobile/ProductNavigation.tsx [hunk 3]
```diff
@@ -111,23 +92,57 @@ export function ProductNavigation({ productsNav, path }: Props) {
...
+                <div className={clsx(styles.subProductContentPage)}>
+                  <div className={styles.header}>
+                    <button key="back" className={styles.back} onClick={() => setProductsSlidePosition("main")}>
+                      <BackArrowIcon />
+                    </button>
+                    <span className={styles.subProductContentTitle}>Resources</span>
+                    <span></span> {/* Spacer */}
+                  </div>
+                  <div className={styles.subProductContent}>
+                    <MegaMenu />
```
Comment (Zelig880) [inline src/components/Header/Nav/ProductNavigation/Mobile/ProductNavigation.tsx:140]: For some reason, the megamenu is not going away when I hover on other menu items or even around it. Maybe this is to do with margin or padding that are making its area bigger than it is.

**Review by Zelig880 — CHANGES_REQUESTED** [review]
Just a couple of improvement needed

File: src/components/Header/Nav/ProductNavigation/Mobile/ProductNavigation.tsx (hunk 3 above)
Comment (markshenouda) [reply src/components/Header/Nav/ProductNavigation/Mobile/ProductNavigation.tsx:140]: I have fixed that by the closing event around it

**Review by Zelig880 — APPROVED** [review]

**Pull request merged**